### PR TITLE
Update stepfunctions troubleshooting doc

### PR DESCRIPTION
**Pull request by kimi-p**
Add two more points for troubleshooting

### CODE REVIEW

File: content/en/serverless/step_functions/troubleshooting.md [hunk 1]
```diff
@@ -34,6 +34,13 @@ If your organization has an existing all-encompassing index with a low limit, pl
 - In your AWS console, open your Step Function and ensure that your state machine has `"Payload.$": "States.JsonMerge($$, $, false)"` on the Lambda steps.
 - Execute your Step Function once and verify that the `TaskScheduled` event log of the Lambda step has the payload containing data from the [Step Function context object][3].
 
+## I can only see `aws.stepfunctions` root span but cannot see any step spans
```
Comment (hestonhoffman) [inline content/en/serverless/step_functions/troubleshooting.md:37]:
```suggestion
## I can see the `aws.stepfunctions` root span but I cannot see any step spans
```

File: content/en/serverless/step_functions/troubleshooting.md [hunk 2]
```diff
@@ -34,6 +34,13 @@ If your organization has an existing all-encompassing index with a low limit, pl
...
+Please enable `Include execution data` option on the state machine's logging. By enabling this option, log execution input, data passed between states, and execution output will be logged in the logs which are then used by Datadog backend to construct these step spans for you.
```
Comment (hestonhoffman) [inline content/en/serverless/step_functions/troubleshooting.md:38]:
```suggestion
Please enable the `Include execution data` option on the state machine's logging. After enabling this option, log execution input, data passed between states, and execution output is logged. The Datadog backend uses the logs to construct these step spans for you.
```

File: content/en/serverless/step_functions/troubleshooting.md [hunk 3]
```diff
@@ -34,6 +34,13 @@ If your organization has an existing all-encompassing index with a low limit, pl
...
+
+## Some step spans are missing in the traces
+- For actions, we support basic actions of Lambda and DynamoDB, e.g. Lambda Invoke, DynamoDB GetItem, DynamoDB PutItem, DynamoDB UpdateItem... etc.
```
Comment (hestonhoffman) [inline content/en/serverless/step_functions/troubleshooting.md:41]:
```suggestion
- For actions, we support basic actions of Lambda and DynamoDB. For example, Lambda Invoke, DynamoDB GetItem, DynamoDB PutItem, DynamoDB UpdateItem and more.
```

File: content/en/serverless/step_functions/troubleshooting.md [hunk 4]
```diff
@@ -34,6 +34,13 @@ If your organization has an existing all-encompassing index with a low limit, pl
...
+- For different flows, we do not support `Wait`, `Choice`, `Map`, `Success`, `Fail`, and `Pass`. For `Parrallel` flow, you would be able to see parallel executing spans stacking on top of each other, but there will be no `Parrallel` spans showing on the flamegraph.
```
Comment (sumedham) [inline content/en/serverless/step_functions/troubleshooting.md:42]:
```suggestion
- For different flows, we do not support `Wait`, `Choice`, `Map`, `Success`, `Fail`, and `Pass`. For `Parallel` flow, you would be able to see parallel executing spans stacking on top of each other, but there will be no `Parallel` spans showing on the flame graph.
```

**Review by sumedham — CHANGES_REQUESTED** [review]

**Review by sumedham — APPROVED** [review]

**Comment by kimi-p** [comment]
@hestonhoffman This is ready to be merged. Could you merge it for me? Thank you!

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**